.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+on:
+  push:
+    branches-ignore:
+      - main
```

Comment:
Same comment as here: https://github.com/saleor/saleor-dashboard/pull/3654/files#r1197647644 - you might need to exclude tags

Reply:
Fixed - thank you 🙇🏻